site/content/getters/blogs.js

```diff
@@ -1,5 +1,12 @@
 import { allBlogs } from "contentlayer/generated";
 
 export default function getBlogs() {
-  return allBlogs.sort((a, b) => new Date(b.created) - new Date(a.created));
+  return allBlogs
+    .map(({ title, description, created, url_path }) => ({
+      title,
+      description,
```

Comment:
```suggestion
description: description || "",
```

Reply:
It's already fixed :)
https://github.com/flowershow/flowershow/commit/1e5a14d3b77e867a142efbd82031a160916a40d7